__mocks__/fs.ts

```diff
@@ -0,0 +1,3 @@
+import { fs } from 'memfs';
+
+module.exports = fs;
```

Comment:
This should be `export = fs`

Do we really need this?

Reply:
I have tried, `export = fs;` does not work, also `export default fs;` does not work.
Yeah, we need it, because third part libraries can use `fs` under the hood, but we need to test some functionality based on third party library. `editor-config.spec.ts` for example